src/app/profile/profile/profile-details/profile-details.component.html

```diff
@@ -19,7 +19,7 @@ <h3>{{ 'PROFIL' | translate }}</h3>
     <button mat-icon-button [app-copy-clipboard]="profileLink" (copied)="notifyCopied()">
       <mat-icon svgIcon="copylink"></mat-icon>
     </button>
-  </div>
+  </h1>
   <div class="detail-label">{{ 'Vorname' | translate }}</div>
   <div class="detail-value">{{ firstname$ | async }}</div>
```

Comment:
These profile `detail-values` seem still to be be a too large to me, compared to the rest. (e.g. in the edit-profile screen)